jekyll/_cci2/workflows.md

```diff
@@ -157,6 +157,33 @@ Following is a screenshot of the Approval dialog box that appears when you click
 
 ![Approval Dialog in On Hold Workflow]({{ site.baseurl }}/assets/img/docs/approval_job_dialog.png)
 
+## Scheduling a Workflow
+
+Workflows that are resource-intensive or that generate reports may be run on a schedule rather than on every commit. This feature is configured by adding a scheduled trigger to the configuration of the workflow. **Note:** The `triggers` key is not yet supported in an installable CircleCI release.
+
+Configure a workflow to run on a set schedule by using the `triggers:` key with the `schedule` option. The `triggers` key is **only** added under your `workflow` key. This feature enables you to schedule a workflow run by using `cron` syntax to represent Coordinated Universal Time (UTC/GMT) for specified branches. 
+
+In the example below, the `nightly` workflow is configured to run every day at 12:00am UTC. The `cron` key is specified using POSIX `crontab` syntax, see the [crontab man page](http://pubs.opengroup.org/onlinepubs/7908799/xcu/crontab.html) for `cron` syntax basics. The workflow will be run on the `master` and `beta` branches. The `commit` workflow has no scheduled trigger configured, so it will run on the push of every commit. 
+
+```
+workflows:
+  version: 2
+  commit:
+    jobs:
+      - test
+      - deploy
+  nightly:
+    triggers:
+      - schedule
+        cron: "0 0 * * *"
+        branches:
```

Comment:
@michelle-luna We need `filter` here too please 🙏

Reply:
got it, thank you!